utilities/physics_tendency_updaters.meta

```diff
@@ -0,0 +1,303 @@
+[ccpp-table-properties]
+  name = apply_tendency_of_x_wind
+  type = scheme
+[ccpp-arg-table]
+  name = apply_tendency_of_x_wind_run
+  type = scheme
+[ nz ]
+  standard_name = vertical_layer_dimension
+  long_name = number of columns
```

Comment:
Wrong long name for this standard name

Reply:
I will fix these (also those I copied from in state_converters, kessler, and kessler_update).